modules/apps/27-interchain-accounts/host/client/cli/query.go

```diff
@@ -85,7 +85,7 @@ func GetCmdPacketEvents() *cobra.Command {
 				return err
 			}
 
-			var resEvents []abci.Event
+			resEvents := make([]abci.Event, 0, len(result.Txs))
```

Comment:
Lets remove this change, this size estimate is wrong, so its better to leave it to golang. Each Tx has many evnets.

Reply:
oh, yeah, good catch!

Reply:
thank you for feedback, i updated